biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/resolve/DependencyResolver.java

```diff
@@ -176,10 +176,14 @@ private void discoverArtifacts(Map<File, ArtifactResult> files, List<DependencyN
 				.getScope())) {
 				continue;
 			}
+
+			List<RemoteRepository> copyOfRepositories = new ArrayList<>(remoteRepositories);
```

Comment:
Not crazy about the var name. `combinedRepositories` would be more meaningful.

Reply:
done